woocommerce-zaver-checkout.php

```diff
@@ -246,19 +246,18 @@ private function __construct() {
 		add_action( 'init', array( $this, 'load_textdomain' ) );
 		add_action( 'before_woocommerce_init', array( $this, 'declare_wc_compatibility' ) );
 
-		$included_settings   = array(
+		$included_settings      = array(
 			array(
 				'type'       => 'title',
 				'is_section' => true,
 			),
 			array( 'type' => 'checkbox' ),
 		);
-		$this->system_report = new SystemReport( 'zaver_checkout', 'Zaver Checkout', $included_settings );
-		$this->logger        = new Logger( 'zaver_checkout', wc_string_to_bool( $settings['logging'] ?? false ) );
-		$this->session       = new Classes\Session();
+		$this->system_report    = new SystemReport( 'zaver_checkout', 'Zaver Checkout', $included_settings );
+		$this->logger           = new Logger( 'zaver_checkout', wc_string_to_bool( $settings['logging'] ?? false ) );
+		$this->session          = new Classes\Session();
```

Comment:
I believe the session class should have its namespace be the same as the other classes in the same folder. I dont believe they start with Classes, but just have the Zaver namespace. In the future when we move everything into the autoloaded PSR-4 standard we will change these anyway so we should keep it the same as it is for the rest of the plugin for now.